pkg/p2p/kadcast/network_test.go

```diff
@@ -1,71 +1,64 @@
 package kadcast_test
 
 import (
-	"bytes"
 	"testing"
 	"time"
 
 	"github.com/dusk-network/dusk-blockchain/pkg/p2p/kadcast"
+	crypto "github.com/dusk-network/dusk-crypto/hash"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	// basePort all listeners derive from
+	basePort = 10000
+	// Number of nodes to bootstrap per a test
+	networkSize = 10
```

Comment:
Generally 2 or 3 bootstrap nodes should be more that enough.

Reply:
NetworkSize is the number of testbed network nodes. I'll change the comment.
Apart from that, maybe I can also decrease the number of bootstrap nodes as they are 9 in the implemented test.

Reply:
But what I mean is that having one two or three of them is more than enough. Theese type of nodes are only useful for sync-in process and should never forward CHUNKS or similar things.